app/config/config.js

```diff
@@ -0,0 +1,57 @@
+"use strict";
+
+const localEnv = _loadLocalEnv(),
+    env = process.env.NODE_ENV || 'development';
+
+
+/**
+ * @property {string} HOST
+ * @property {string} PORT
+ * @property {string} MONGODB_URL
+ */
+const config = {
+
+    development: {
+        HOST: localEnv.HOST || '188.166.63.248',
+        PORT: localEnv.PORT || '9030',
+
+        MONGODB_URL: localEnv.MONGODB_URL || 'mongodb://localhost:27017/differ-canvas'
+    },
+
+    test: {
+        HOST: '127.0.0.1',
+        PORT: '3000',
+
+        MONGODB_URL: 'mongodb://localhost:27017/differ-canvas_test_integration'
+    },
+
+    production: {
+        HOST: '127.0.0.1',
+        PORT: '9010',
+
+        MONGODB_URL: 'mongodb://localhost:27017/differ-canvas'
+    }
+};
+
+
+module.exports = config[env];
+
+
+/**
+ * @type function
```

Comment:
Got the point. You prefer 'easier to spot /alter mistake', and I prefer 'keep it dry'.
Will stick to your one.
validateBody(req.body, {
  name: Joi.string().max(40).required(),
  email: Joi.string().email().required(),
 ...
});
Like this?